### PR TITLE
MDL-61301 tool_policy: add capability to view acceptances

### PR DESCRIPTION
**Pull request by marinaglancy**

### CODE REVIEW

File: admin/tool/policy/editpolicydoc.php [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 admin_externalpage_setup('tool_policy_managedocs', '', ['policyid' => $policyid, 'versionid' => $versionid],
     new moodle_url('/admin/tool/policy/editpolicydoc.php'));
+require_capability('tool/policy:managedocs', \context_system::instance());
```
[the remark this answers is not in the document]
Comment (mudrd8mz) [reply admin/tool/policy/editpolicydoc.php]: See settings.php where this admin page is defined, it has the required capability declared there.

[the remark this answers is not in the document]
Comment (marinaglancy) [reply admin/tool/policy/editpolicydoc.php]: I changed settings.php, this page requires either capability managedocs or viewacceptances now

[the remark this answers is not in the document]
Comment (mudrd8mz) [reply admin/tool/policy/editpolicydoc.php]: Oh I see - now that we have more caps added, this is to make sure that the user has this one always right?

[the remark this answers is not in the document]
Comment (marinaglancy) [reply admin/tool/policy/editpolicydoc.php]: on page managedocs.php - either of these two caps
on page editpolicydoc.php - only managedocs
on page acceptances.php - only viewacceptances

[the remark this answers is not in the document]
Comment (mudrd8mz) [reply admin/tool/policy/editpolicydoc.php]: Yup, I commented prematurely without checking the rest of the patch.

**Pull request merged**